src/librustc_mir/interpret/step.rs

```diff
@@ -64,29 +64,28 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
 
         if let Some(stmt) = basic_block.statements.get(stmt_id) {
             assert_eq!(old_frames, self.cur_frame());
-            self.statement(stmt)?;
+            self.eval_stmt(stmt)?;
```

Comment:
I am not opposed to renaming these, but then please
* keep the long form, `statement`, as it is more readable
* be consistent: if you make it `eval_statement`, it should also be `eval_terminator`.

Reply:
> I am not opposed to renaming these, but then please
> 
>     * keep the long form, `statement`, as it is more readable
> 
>     * be consistent: if you make it `eval_statement`, it should also be `eval_terminator`.

Thank you for your feedback :+1:  I can change the function name from `fn eval_stmt()` to `fn eval_statement()`, 
but there's an issue with renaming `fn handle_terminator()` to `fn eval_terminator()`; 
there's already another method [`fn eval_terminator()`](https://github.com/rust-lang/rust/blob/660326e9791d5caf3186b14521498c2584a494ab/src/librustc_mir/interpret/terminator.rs#L14) defined for `InterpCx` and the build fails..

Currently `fn handle_terminator()` calls `fn eval_terminator()`.

Reply:
Hm, good point.

But `handle_terminator` is not really any more clear than `terminator`, I feel... and these are both private functions so it is rather unlikely there will be any confusion.

Reply:
I tried building the project after renaming `fn handle_terminator()` to `fn eval_terminator()`, but the build failed (due to two methods for `InterpCx` having same names). 

Since `fn handle_terminator()` was a short function that internally calls 
`fn eval_terminator()`, I pushed a new commit that merges the logic inside
`fn handle_terminator()` to the existing `fn eval_terminator()`. 
(2 functions meged to one; no more name conflicts)

Reply:
`eval_terminator` is already a monster function though, I'd rather not make it any bigger.

Also,your change is actually incorrect: `eval_terminator` contains some inner `return` that would now skip the code you moved to the end of that function.